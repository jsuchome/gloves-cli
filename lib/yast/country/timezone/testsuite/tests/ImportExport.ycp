/**
 * tests for Timezone.ycp autoyast functions
 * Author:	jsuchome@suse.cz
 * $Id: ImportExport.ycp 32769 2006-09-08 09:29:15Z jsuchome $
 */
{
    // testedfiles: Timezone.ycp

    import "Testsuite";

    map READ = $[
	"target" : $[
	    "size"		: 1,
	    "dir"		: [],
	    "yast2"		: $[],
	],
    ];

    map E	= $[
	"target" : $[
	    "bash_output"	: $[],
	],
    ];

    import "Mode";
    Mode::SetMode ("autoinst_config");

    Testsuite::Init ([READ,$[],E], nil);

    import "Timezone";

    Testsuite::Test (``(Timezone::Export ()), [], nil);

    Testsuite::Test (``(Timezone::Modified ()), [], nil);

    map imported	= $[
	"timezone"	: "Europe/Berlin"
    ];

    Testsuite::Test (``(Timezone::Import (imported)), [$[],$[],E], nil);

    Testsuite::Test (``(Timezone::Modified ()), [], nil);

    Testsuite::Test (``(Timezone::Export ()), [], nil);

    Testsuite::Test (``(Timezone::Summary ()), [], nil);
}
