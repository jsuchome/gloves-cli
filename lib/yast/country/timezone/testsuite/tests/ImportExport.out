Return	$["hwclock":"UTC", "timezone":""]
Return	false
Read	.target.yast2 "timezone_raw.ycp" nil
Return	true
Return	true
Return	$["hwclock":"UTC", "timezone":"Europe/Berlin"]
Return	<ul><li>Current Time Zone:  / Europe/Berlin</li><li>Hardware Clock Set To UTC</li></ul>
